Mathlib/CategoryTheory/Filtered/Final.lean

```diff
@@ -427,6 +428,24 @@ end Pi
 
 section Prod
 
+open IsFiltered in
+instance Functor.final_prod' [IsFilteredOrEmpty C] {E : Type u₃} [Category.{v₃} E] (F : C ⥤ D)
```

Comment:
Instead of this approach, would it be possible to show that in general `Functor.prod` of two final functors is final (through the study of `StructuredArrow` categories of a product functor), and then use that `Functor.prod'` is obtained by composing with the diagonal functor `C ⥤ C × C`, which is final (this holds more generally for sifted (or empty) categories).

Reply:
Done!